python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -15,6 +15,8 @@
 from ml_wrappers.model.pytorch_wrapper import WrappedPytorchModel
 from ml_wrappers.model.wrapped_classification_model import \
     WrappedClassificationModel
+from torchvision import transforms as T
+from vision_explanation_methods.explanations import common as od_common
```

Comment:
Curious if we should indeed have this as required dependency and/or trap the ImportError here as well. CC @imatiach-msft